llvm/lib/Transforms/Scalar/RewriteStatepointsForGC.cpp

```diff
@@ -1423,7 +1423,7 @@ static constexpr Attribute::AttrKind FnAttrsToStrip[] =
 
 // Create new attribute set containing only attributes which can be transferred
 // from original call to the safepoint.
-static AttributeList legalizeCallAttributes(LLVMContext &Ctx,
+static AttributeList legalizeCallAttributes(LLVMContext &Ctx, unsigned NumArgs,
                                             AttributeList OrigAL,
                                             AttributeList StatepointAL) {
```

Comment:
Looks like it can be easier/cleaner to just pass CallInst here? (maybe together with IsMemIntrinsic flag and build OrigAl inside this function)